### PR TITLE
Repackaging + fixing samples

### PR DESCRIPTION
**Pull request by artursouza**
# Description

Took PR from brunoborgers (https://github.com/dapr/java-sdk/pull/14) and did the following:
* Changed code from Java 11 to Java 8.
* Fixed examples.
* Fixed build error when proto were already generated, then the 2nd compile would fail.
* Renamed `client` to `sdk` since we will be having a single jar for everything except examples at this point. Later on, we can refactor into new packages if needed.
* Updated gitignore.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #16

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [X] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: examples/src/main/java/io/dapr/examples/Example.java [hunk 1]
```diff
@@ -2,27 +2,21 @@
...
+import io.dapr.DaprProtos.*;
 import io.grpc.ManagedChannel;
 import io.grpc.ManagedChannelBuilder;
 
 /**
  * Simple example, to run:
- *   mvn compile
- *   dapr run --grpc-port 50001 -- mvn exec:java -Dexec.mainClass=io.dapr.examples.Example
+ *   mvn clean install
+ *   dapr run --grpc-port 3000 -- mvn exec:java -pl=examples -Dexec.mainClass=io.dapr.examples.Example
```
Comment (yaron2) [inline examples/src/main/java/io/dapr/examples/Example.java]: Why is this changed to port `3000`?

File: examples/src/main/java/io/dapr/examples/OrderManager.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+/**
+ * OrderManager web app.
+ * 
+ * Based on the helloworld Node.js example in https://github.com/dapr/samples/blob/master/1.hello-world/app.js
+ *
+ * To install jars into your local maven repo:
+ *   mvn clean install
+ *
+ * To run (after step above):
+ *   dapr run --app-id orderapp --app-port 3000 --port 3500 -- mvn exec:java -pl=examples -Dexec.mainClass=io.dapr.examples.OrderManager
```
Comment (yaron2) [inline examples/src/main/java/io/dapr/examples/OrderManager.java]: If the app port here is 3000, the line:
```
 *   dapr run --grpc-port 3000 -- mvn exec:java -pl=examples -Dexec.mainClass=io.dapr.examples.Example
```

Will cause a port collision

File: examples/src/main/java/io/dapr/examples/Example.java (hunk 1 above)
Comment (artursouza) [reply examples/src/main/java/io/dapr/examples/Example.java]: To look like the other samples.

File: examples/src/main/java/io/dapr/examples/OrderManager.java (hunk 2 above)
Comment (artursouza) [reply examples/src/main/java/io/dapr/examples/OrderManager.java]: Sorry, fixing it.

Comment (artursouza) [reply examples/src/main/java/io/dapr/examples/OrderManager.java]: Fixed the other one to be something else.

File: examples/src/main/java/io/dapr/examples/Example.java (hunk 1 above)
Comment (artursouza) [reply examples/src/main/java/io/dapr/examples/Example.java]: Fixed to the old number.

File: README.md [hunk 3]
```diff
@@ -5,22 +5,32 @@ This is the Dapr SDK for Java, based on the auto-generated proto client.<br>
 For more info on Dapr and gRPC, visit [this link](https://github.com/dapr/docs/tree/master/howto/create-grpc-app).
 
 ### Installing
+
+The client will generate gRPC Client and Protobuf classes during build.
```
Comment (amanbha) [inline README.md]: Could you clarify this comment, its not very clear to me.

File: README.md [hunk 4]
```diff
@@ -5,22 +5,32 @@ This is the Dapr SDK for Java, based on the auto-generated proto client.<br>
 For more info on Dapr and gRPC, visit [this link](https://github.com/dapr/docs/tree/master/howto/create-grpc-app).
 
 ### Installing
+
+The client will generate gRPC Client and Protobuf classes during build. 
+
+Make sure to clone this repository including the submodules:
```
Comment (amanbha) [inline README.md]: We are not using any submodules in this repo.
